sonar-plugin-api/src/main/java/org/sonar/api/server/authentication/UserIdentity.java

```diff
@@ -151,16 +159,25 @@ public Builder setEmail(@Nullable String email) {
 
     /**
      * Set groups of the user.
```

Comment:
I would mention the term "membership" in the javadoc, for instance "Set group membership".